src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -291,11 +292,11 @@ private static ExtBidRequest requestExt(BidRequest bidRequest) {
     private List<BidderRequest> makeBidderRequests(List<String> bidders, BidRequest bidRequest,
                                                    Map<String, String> uidsBody, UidsCookie uidsCookie,
                                                    ObjectNode userExtNode, ExtRegs extRegs, Map<String, String> aliases,
-                                                   List<Imp> imps, Map<Integer, Boolean> vendorsToGdpr) {
+                                                   List<Imp> imps, GdprResponse gdprResponse) {
```

Comment:
m.b. it is better to leave `Map<Integer, Boolean> vendorsToGdpr` argument since we don't use any other data from `GdprResponse` model.